vit-servicing-station-tests/src/tests/bootstrap/parameters.rs

```diff
@@ -29,7 +35,8 @@ pub fn in_settings_file_does_not_exist() {
         .in_settings_file(&non_existing_file)
         .build()
         .assert()
-        .failure();
+        .failure()
+        .code(ApplicationExitCode::LoadSettingsError as i32);
```

Comment:
`into()` cannot be used here since the assert takes the type from the function parameter apparently. `Into<i32>` should be used, but for that case, it is more straight forward to use the `as` I think.